flixel/system/frontEnds/DebuggerFrontEnd.hx

```diff
@@ -155,8 +159,11 @@ class DebuggerFrontEnd
 			FlxG.stage.stageFocusRect = false; // don't show yellow focus rect on flash
 			FlxG.stage.focus = FlxG.game;
 		}
+		
+		visible = Value;
+		visibilityChanged.dispatch();
 		#end
 		
-		return visible = Value;
```

Comment:
This now leads to `set_visible()` not doing anything with `FLX_NO_DEBUG`. I think we should be consistent with the behavior of `drawDebug` here again.